pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -368,14 +387,28 @@ func (c *kvEventToRowConsumer) ConsumeEvent(ctx context.Context, ev kvevent.Even
 	a := ev.DetachAlloc()
 	a.AdjustBytesToTarget(ctx, int64(len(keyCopy)+len(valueCopy)))
 
-	if err := c.sink.EmitRow(
-		ctx, topic, keyCopy, valueCopy, schemaTimestamp, mvccTimestamp, a,
-	); err != nil {
-		return err
-	}
-	if log.V(3) {
-		log.Infof(ctx, `r %s: %s -> %s`, updatedRow.TableName, keyCopy, valueCopy)
+	e := sinkEvent{
+		topic:   topic,
+		key:     keyCopy,
+		value:   valueCopy,
+		updated: schemaTimestamp,
+		mvcc:    mvccTimestamp,
+		alloc:   a,
+	}
```

Comment:
this sinkEvent will definitely result in an allocation (which escapes to the heap)...  It might make sense to have an sync.Pool of *sinkEvent so that those can be reused...

Reply:
Done